rust/item/Cargo.toml

```diff
@@ -21,3 +21,4 @@ arrayref = "0.3.6"
 spl-associated-token-account = { version="1.0.3", features = [ "no-entrypoint" ] }
 spl-token = { version="3.1.1", features = [ "no-entrypoint" ] }
 metaplex-token-metadata = { version="0.0.1", features = [ "no-entrypoint" ] }
+raindrops-namespace = { features = ["no-entrypoint", "cpi"], path = "../namespace"}
```

Comment:
Why is this the main namespace crate instead of a namespace-cpi crate? It seems if we were to take this approach, we shouldn’t use the main crates in any of the other programs.

Reply:
Yeah totally could be the cpi crate as well, just did this real quick